### ISSUE TITLE
OperationID API schema attribute support

### ISSUE THREAD

**Issue by isolationism** [vitalik/django-ninja#54]
I'm a new user to your library — just discovered on Reddit a few days ago. Apologies if this was answered elsewhere.

OpenAPI supports a parameter called `operationId`, which provides an API-unique discriminator for an endpoint. There is annoyingly no hotlink to the relevant section; OpenAPI's docs could use improving:

https://swagger.io/docs/specification/paths-and-operations/

The ID isn't particularly useful for humans (it doesn't appear at all in the docs), but for a programmable client it is particularly useful as it avoids all the clutter of needing to deal with URL templates, method, and headers — you can just provide the operationId along with any arguments the endpoint requres. 

If it's not too bold to point out, I use these today in [FastAPI](https://fastapi.tiangolo.com/advanced/path-operation-advanced-configuration/) and would love to see — or bring — this support in/to django-ninja, if there's enough interest (and maybe a little help; I'm not a fantastic coder).

**Comment by vitalik** [comment]
Hi @isolationism 
as I understand  you need define manually operation ids ?

something like this: 
```
@api.get("/items/", operation_id="your_custome_name")
```

this is actually in my backlog - should be released shortly

**Comment by isolationism** [comment]
That is my request precisely, yes. Thank you!

**Comment by vitalik** [comment]
Done, please try latest :

`pip install -U django==0.9.5`

**Comment by isolationism** [comment]
Thank you — I installed `django-ninja==0.9.7` today and confirm it works exactly as expected in three cases:

— Auto-generated ID (underscore-separated classpath from project root)
— `operation_id` specified as decorator method argument
— By implementing `get_openapi_operation_id()` (I used a uuid4 for now). 

I particularly appreciate the third case as it gives the ability to write our own router/url resovler; I for one would like to explore the possibility of integrating with some of Django's built-in URL libraries for reverse lookups etc.

Thanks for the swift PR and great communication, love where this project is going.
